src/widgets/splits/SplitInput.hpp

```diff
@@ -109,6 +137,7 @@ private slots:
     void editTextChanged();
 
     friend class Split;
+    friend class ReplyThreadPopup;
```

Comment:
Is it needed here?

Reply:
This is necessary for `ReplyThreadPopup` to be able to call `giveFocus` on the SplitInput